ui/app/AppLayouts/Communities/CommunitiesPortalLayout.qml

```diff
@@ -137,16 +137,17 @@ StatusSectionLayout {
                     onClicked: Global.importCommunityPopupRequested()
                 }
 
-                StatusButton {
+                 StatusButton {
                     id: createBtn
+                    visible: communitiesStore.createCommunityEnabled || communitiesStore.testEnvironment
```

Comment:
I just found out that the new e2e tests don't have this flag enabled now. Is there another flag I can use to show this button in tests? cc @anastasiyaig 

Otherwise, we need to tweak the communities tests to set the setting to ON before creating the community and it seems like effort for nothing

Reply:
easiest solution now is to exclude the test from the run (which i already did). We need to fix the flow and i will be done ASAP and i will bring the test back when it is done